src/class/IContentTypes/IContentTypes.cpp

```diff
@@ -0,0 +1,126 @@
+#include <IContentTypes/IContentTypes.hpp>
+
+std::map<std::string, std::string> IContentTypes::create_map()
```

Comment:
can be static since it is only relevant in this file

**static** std::map<std::string, std::string> IContentTypes::create_map()

Reply:
great idea!